### PR TITLE
Update URLs and commands from hg/Bitbucket to git/Github

### PR DESCRIPTION
**Pull request by diazona**
This change updates remaining URLs and commands that reference Mercurial and/or Bitbucket to their corresponding Git/Github versions.

I had this branch ready for a while and meant to push it earlier but I had forgotten.

### CODE REVIEW

**Review by coordt — APPROVED** [review]
looks good

File: README.rst
Comment (redtoad) [inline README.rst:203]: We should replace this line with a link to the Github project page .

Comment (diazona) [reply README.rst:203]: Oooh good catch thanks

Comment (diazona) [reply README.rst]: Fixed in the latest push

**Comment by diazona** [comment]
I grepped for `bitbucket` and `hg` (as a full word) and found no further references that need to be changed, but do let me know if I missed anything else. Otherwise, I think this is ready once you recheck it @redtoad, so feel free to merge it yourself if you want to.

**Review by coordt — APPROVED** [review]

**Comment by diazona** [comment]
Hey @redtoad just wondering if you wanted to follow up on your earlier comment before this gets merged? I'd like to wrap this up soon and unblock us from making the new release.

**Comment by redtoad** [comment]
I've rebased the branch and will merge in 5...4...3...2...1...

**Comment by diazona** [comment]
Thanks!

**Pull request merged**
